lib/videojs-playlist-ui.js

```diff
@@ -4,6 +4,10 @@
 
 import videojs from 'video.js';
 
+// support VJS5 & VJS6 at the same time
+const Dom = videojs.dom || videojs;
```

Comment:
hmmm I'm not sure. Internally its `Dom`. We expose it as `dom`. What do you think @gkatsev?